drivers/gpu/drm/vc4/vc4_hdmi.c

```diff
@@ -917,6 +917,50 @@ static void vc4_hdmi_csc_setup(struct vc4_hdmi *vc4_hdmi,
 	HDMI_WRITE(HDMI_CSC_CTL, csc_ctl);
 }
 
+
+/*
+ * If we need to output Full Range RGB, then use the unity matrix
+ *
+ * [ 1      0      0      0]
+ * [ 0      1      0      0]
+ * [ 0      0      1      0]
+ *
+ * Matrix is signed 2p13 fixed point, with signed 9p6 offsets
```

Comment:
Need to check those descriptions. At least one of the CCMs had a very strange representation for negative values, but that may have been the ISP.

Reply:
Just to resolve my own comment, it's the CSC coefficients in the dlist that I was thinking of. HDMI block is straightforward.